packages/react-scripts/scripts/init.js

```diff
@@ -125,7 +119,7 @@ module.exports = function(
 
   const templatePackage = templateJson.package || {};
 
-  // TODO: Deprecate support for root-level `dependencies` and `scripts` in v4.
+  // TODO: Deprecate support for root-level `dependencies` and `scripts` in v5.
```

Comment:
We didn't get a minor out with a warning for this, so we will leave it in for v4.